mitosheet/src/mito/components/elements/Toggle.tsx

```diff
@@ -47,9 +52,14 @@ const Toggle = (props: ToggleProps): JSX.Element => {
 
     const disabled = props.disabled === true
     const className = props.className !== undefined ? props.className : ''
+    const title = props.title !== undefined ? props.title : undefined
```

Comment:
I think this is functionally equivalent to just using `props.title` without the ternary operator